.vib/grafana-loki/cypress/cypress/integration/grafana_loki_spec.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+import body from '../fixtures/logs-body.json';
```

Comment:
Is this the right way to import a fixture? Have you tried using `cy.fixture` or even `cy.readFile`?

Reply:
Yes, I've tried both for this case as far as I remember. 
In this case, I need to use the whole file and insert it into a body and AFAIK this is the only way.